Cargo.lock

```diff
@@ -1839,6 +1930,28 @@ dependencies = [
  "tiny-keccak",
 ]
 
+[[package]]
+name = "ethereum-consensus"
+version = "0.1.0"
+source = "git+https://github.com/ralexstokes/ethereum-consensus#f36ebe9d76e752ee6acf29250da5c7b63ea56784"
+dependencies = [
+ "async-stream",
+ "blst",
+ "enr",
+ "hex",
+ "integer-sqrt",
+ "libp2p-core 0.32.1",
```

Comment:
With that PR merged I think a `cargo update` will sort this out, as `mev-rs` doesn't pin an `ethereum-consensus` version

Reply:
Looks like it worked! https://github.com/sigp/lighthouse/pull/3134/commits/5ad7632fc8481882b9367a118ae4b2f3c8c9d053#diff-13ee4b2252c9e516a0547f2891aa2105c3ca71c6d7a1e682c69be97998dfc87eL1944

Reply:
Heck yeah!

Reply:
Ah dang there's actually a knock-on effect of this because `mev-build-rs` depends on `beacon-client-api` which used `PeerId`. I'm trying to update `beacon-client-api` here: https://github.com/ralexstokes/beacon-api-client/pull/28/files

Reply:
ok it looks all good now